src/main/kotlin/com/terraformation/backend/i18n/Messages.kt

```diff
@@ -102,7 +105,28 @@ class Messages {
           title = "$deviceName cannot be detected.",
           body = "$deviceName cannot be detected. Please check on it.")
 
+  fun historyAccessionCreated() = "created accession"
+
+  fun historyAccessionStateChanged(newState: AccessionState) =
+      "updated the status to ${newState.displayName}"
+
+  /**
+   * Returns the full name of a user based on their first and last names. It's possible for users to
+   * not have first or last names, e.g., if they were created by being added to an organization and
+   * haven't gone through the registration flow yet; returns null in that case. If the user has only
+   * a first name or only a last name, returns whichever name exists.
+   */
+  fun userFullName(firstName: String?, lastName: String?): String? =
```

Comment:
would using email be a fallback option?

Reply:
The PRD just says to include full name. One hesitation I have with putting email in there too is a permission question: currently, you can have permission to view accession data but not permission to view the organization's user list. Exposing people's email addresses to non-admin users probably needs to be a product decision.

Reply:
good point, also may have eula or service terms disclosure implications. i keep hearing that there might be one signed in account that multiple users use.. maybe this won't be an issue after all.. history may show that one signed in user most of the time.